io/shared/src/main/scala/fs2/io/process/Process.scala

```diff
@@ -34,11 +34,14 @@ sealed trait Process[F[_]] {
 
   /** A `Pipe` that writes to `stdin` of the process. The resulting stream should be compiled
     * at most once, and interrupting or otherwise canceling a write-in-progress may kill the process.
+    * Failure to write promptly to this `Stream` when required can may cause the process to block, or
+    * even deadlock.
     */
   def stdin: Pipe[F, Byte, Nothing]
 
   /** A `Stream` that reads from `stdout` of the process. This stream should be compiled at most once,
-    * and interrupting or otherwise canceling a read-in-progress may kill the process.
+    * and interrupting or otherwise canceling a read-in-progress may kill the process. Not draining
+    * this `Stream` can block the process and may cause the process to block, or even deadlock.
```

Comment:
Same is true of `stderr`.

Reply:
I thought that but was not sure because the docs on the java side only mention stdin and stdout:

> Because some native platforms only provide limited buffer size for standard input and output streams, failure to promptly write the input stream or read the output stream of the subprocess may cause the subprocess to block, or even deadlock.

Reply:
Oh. Maybe I'm wrong actually and `stderr` is special. Thanks for pointing it out.

Reply:
I'm not entirely sure it's special, so I've added the warning to `stderr` as well.